thirdparty/kyaml/runfn/runfn.go

```diff
@@ -438,27 +443,30 @@ func (r *RunFns) defaultFnFilterProvider(spec runtimeutil.FunctionSpec, fnConfig
 				AllowMount: true,
 			},
 		}
-		cf := &runtimeutil.FunctionFilter{
+		fltr = &runtimeutil.FunctionFilter{
```

Comment:
I would like to get away from crafting this filter here manually but it will be easier once my structured result changes are merged in.